docs/Demo_Projects/Tracking_tutorial.md

```diff
@@ -0,0 +1,388 @@
+    ```
+    west build -b thingy91_nrf9160ns -p -t menuconfig
+    ``` 
+    
+    A config screen will open:
+
+    - Open the folder: `anjay-zephyr-client --->`
+        - Select: `Enable manual requests for cell-based location`
+        - Open the folder: `Enable GPS on nRF9160-based devices --->`
+            - Select `Enable A-GPS using Nordic Location Services over LwM2M`
+        
+
+    ![menuconfig](images/menuconfig1.png "Anjay menuconfig")
+
+    After making the configuration changes, close the config menu by pressing `Q` and save it by pressing the key `Y`.
+
+    **Build the project** using the updated configuration by running:
+    ```
+    west build
+    ```
+
+0. Find the `app_signed.hex` file under the `build/zephyr` directory in the project folder.
+
+### Write the firmware to the Thingy:91
+
+- When using the Thingy:91, use the `app_signed.hex` file which you can find in the `build/zephyr` directory.
+- Flash it using **Programmer** application in **nRF Cloud for Desktop** via **MCUboot**. 
+
+    *For more information on flashing the Thingy:91 using MCU Boot, see [link](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_thingy91_gsg.html#program-the-nrf9160-sip-application)*
+
+![nrf - write.png](images/nrf-write_done.png)
+
+- Powercycle the Thingy:91 to activate the application.
+
+
+## Connect the Thingy:91 to Coiote IoT Device Management
+
+To connect the board:
+
+1. [Log in](https://eu.iot.avsystem.cloud/) to Coiote DM and from the left side menu, select **Device Inventory**.
+1. In **Device Inventory**, click **Add device**.
+1. Select the **Connect your LwM2M device directly via the Management server** tile.
+    
+    ![Add via Management server](https://iotdevzone.avsystem.com/docs/LwM2M_Client/Nordic/images/mgmt_tile.png)
+
+1. In the **Device credentials** step:
+    - Think of an unique **Endpoint name**.
+    - **Key Identity** is the same as the Endpoint name
+    - Create a **Key** and store this somewhere to retrieve later when configuring your device.
+    - Click the **Add device** button and click **Confirm** in the confirmation pop-up.
+
+    ![Add Management quick](https://iotdevzone.avsystem.com/docs/LwM2M_Client/Nordic/images/add_mgmt_quick.png)
+
+## Configure the Client
+
+0. With the Thingy:91 still connected to a serial port interface, connect to your device using a serial communication program (e.g. Minicom, RealTerm or PuTTY).
+
+0. Use the `anjay` command to list possible options:
+
+    ```
+    uart:~$ anjay
+    anjay - Anjay commands
+    Subcommands:
+    start   :Save config and start Anjay
+    stop    :Stop Anjay
+    config  :Configure Anjay params
+    ```
+
+    !!! tip
+        To show available subcommands, use the **Tab** key.
+
+0. Check your default credentials by following the instructions in the program:
+
+    ```
+    anjay config show
+    ```
+
+    ![Anjay configuration](images/anjay_config.png "Anjay configuration")
+
+0. Update your device credentials:
```

Comment:
`Update your device credentials by running the following commands:` and remove "run the following command" from lines 144 and 150